src/Common/src/System/Net/Logging/GlobalLog.cs

```diff
@@ -253,7 +253,7 @@ public static void Dump(byte[] buffer, int offset, int length)
                 return;
             }
 
-            var bufferSegment = new byte[length];
```

Comment:
Please don't change this. The original code is in line with the style guidelines.